src/hooks/api/queries/useNotificationCountQuery.ts

```diff
@@ -1,13 +1,15 @@
 import { useQuery } from '@tanstack/react-query'
 import { getNotificationCount } from '@/services/notification/notification'
 
-const useNotificationCountQuery = () => {
+const useNotificationCountQuery = ({ isLoggedIn }: { isLoggedIn: boolean }) => {
   return useQuery({
     queryKey: ['notificationCount'] as const,
     queryFn: () => {
```

Comment:
```suggestion
    queryFn: () =>  getNotificationCount()
```
따로 res를 return 하지 않고 위 코드로 바꿔도 훨씬 깔끔할 것 같네요!